pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1227,6 +1242,22 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 		return
 	}
 
+	//TODO: mountPoints must be fill with target paths on the container once we decide UI to domainmgr flow for vol mount.
+	mountPoints := make([]MountPoints, 0)
+	mpFileName := mountPointFileName(config.AppNum)
+	mpFile, err := os.Create(mpFileName)
+	if err != nil {
+		log.Fatal("os.Create for ", mpFileName, err)
+	}
+	defer mpFile.Close()
+
+	if err := writeMountPointsToFile(mountPoints, *status, mpFile); err != nil {
+		log.Errorf("Failed to create mount point file. %v", err.Error())
+		status.LastErr = fmt.Sprintf("%v", err)
+		status.LastErrTime = time.Now()
+		return
+	}
+
```

Comment:
It is better to spin this off into a separate function and call from here?
    doCreateMountPointsFileForApp(cfg)

This function ( doActivate ) is already quite big.

Reply:
Updated